canvas/models.py

```diff
@@ -336,4 +336,4 @@ def revert(self):
             course_reg.canvas_user_id: {
                 'posted_grade': 0,
             }
-        })
+        })
```

Comment:
roll back

Reply:
I am not able to understand what is the reason for the rollback here? Do you want me to add an additional line would that do it?